.github/scripts/launch-manager-instance.py

```diff
@@ -21,7 +21,7 @@ def main():
         sys.exit(0)
 
     print("Launching a fresh manager instance. This will take a couple minutes")
-    awstools.main([
+    awstools.awstools.main([
```

Comment:
It has to be because we've got `deploy/awstools/awstools.py` and I didn't want to `import awstools.awstools as awstools` or change `deploy/awstools/__init__.py` to import everything from `awstools.py`.

I really think that we may end up wanting to rename `deploy/awstools` to `deploy/platformtools` as we start to add other modules for different platforms but I've been waiting to propose that until we have more code to organize.